dashbuilder/dashbuilder-shared/dashbuilder-js/packages/logo-component/webpack.config.js

```diff
@@ -20,18 +20,23 @@ const { merge } = require("webpack-merge");
 const common = require("../../webpack.common.config");
 
 module.exports = async (env, argv) => {
+  let entryPoint = "./src/index.tsx";
+  const copyResources = [
+    { from: "./static/images", to: "./images" },
+    { from: "./static/index.html", to: "./index.html" },
+    { from: "./static/manifest.json", to: "./manifest.json" }
+  ];
+
+  if (process.env.WEBPACK_DEV_SERVER) {
+    entryPoint = "./src/index-dev.tsx";
+    copyResources.push({ from: "./static/manifest.dev.json", to: "./manifest.dev.json" });
+  }
```

Comment:
```suggestion
  if (process.env.WEBPACK_DEV_SERVER) {
    entryPoint = "./src/index-dev.tsx";
    copyResources.push({ from: "./static/manifest.dev.json", to: "./manifest.dev.json" });
  } else {
     entryPoint = "./src/index.tsx";
  }
```

Reply:
That's what I meant.. nothing required, though, just a matter of style, probably.